src/org/usfirst/frc4904/robot/commands/IndexerGrabberSet.java

```diff
@@ -0,0 +1,25 @@
+package org.usfirst.frc4904.robot.commands;
+
+
+import org.usfirst.frc4904.robot.RobotMap;
+import edu.wpi.first.wpilibj.command.Command;
+
+public class IndexerGrabberSet extends Command {
+	protected final boolean clasped;
+
+	public IndexerGrabberSet(boolean clasped) {
+		super(clasped ? "IndexerGrabberClasp" : "IndexerGrabberRelease");
+		requires(RobotMap.Component.rollyBOIGrabber);
+		this.clasped = clasped;
+	}
+
+	@Override
+	protected void initialize() {
+		RobotMap.Component.rollyBOI.grabber.set(clasped);
+	}
+
+	@Override
+	protected boolean isFinished() {
+		return false;
```

Comment:
i'm pretty sure we determined this had to be true

Reply:
@ajnadel said false